pkg/syncer/metadatasyncer.go

```diff
@@ -38,6 +38,8 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/common/commonco/k8sorchestrator"
```

Comment:
Can we move these lines to the next block of imports?

Reply:
done